src/components/MainComponents/MainCSS.js

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable */
+
+import React from 'react'
+import styled, { css } from 'styled-components'
+import { Intro } from 'components/MainComponents/Intro'
+import { Skills } from 'components/MainComponents/Skills'
+import { Projects } from 'components/MainComponents/Projects'
+import { Tech } from 'components/MainComponents/Tech'
+import { Thoughts } from 'components/MainComponents/Thoughts'
+import { ForMore } from 'components/MainComponents/ForMore'
+
+export const Main= () => {
```

Comment:
The MainCSS is really interesting! You could have some of this moved to global styles to fill that component up and then ommit this one (or vice versa!). However, please don't take it as a critique, it's just a another way of solving this. I like how easily the *props* are passed in different background colors, really nice! (personally, I made a twin of my section, didn't realise you could pass THIS MANY props in one. Amazing!)